### PR TITLE
feat(init): Add stryker init command

### LINKED ISSUE #1558: Generate filled stryker config file with cli command
**Is your feature request related to a problem? Please describe.**
I would be nice if stryker can generate a complete stryker config file with all defaults filled in.

**Describe the solution you'd like**
`dotnet stryker init` generates a config file with all defaults. No stryker run is started.
If a stryker config already exists, it is overwritten.

### PR DESCRIPTION
**Pull request by ysbakker**
Add stryker init command:
- Creates a config file with default values by default
- Allows overriding config values with command line options

Closes #1558

### CODE REVIEW

File: src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLIInitCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    private readonly Mock<ILoggingInitializer> _loggingInitializerMock = new();
+
+    public StrykerCLIInitCommandTests()
+    {
+        _target = new StrykerCli(_strykerRunnerMock.Object, null, _loggingInitializerMock.Object, _nugetClientMock.Object);
+    }
+
+    [Theory]
+    [InlineData("init")]
+    [InlineData("init", "--config-file", "\"test.json\"")]
```
Comment (richardwerkman) [inline src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLIInitCommandTests.cs:23]: Will the command create a file named `test.json` in this case?

Comment (ysbakker) [reply src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLIInitCommandTests.cs:23]: Yes. Currently writing integration tests to assert this 😄

File: azure-pipelines.yml [hunk 2]
```diff
@@ -194,6 +194,11 @@ stages:
       parameters:
         testName: 'windows-netframework'
         workingDirectory: 'integrationtest/TargetProjects/NetFramework/FullFrameworkApp.Test'
+    - template: pipeline-templates/run-integration-test-steps.yml
+      parameters:
+        testName: 'windows-initcommand'
+        workingDirectory: 'integrationtest/TargetProjects'
+        strykerCommands: init --config-file "InitCommand/test-config.json" -p "TestProject.csproj"
```
Comment (richardwerkman) [inline azure-pipelines.yml:201]: I'd like to see 2 tests, 1 with default values, and one that overrides all possible values

File: src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs [hunk 3]
```diff
@@ -97,31 +97,31 @@ public void ShouldDisplayLogo()
...
-
-            var target = new StrykerCli(mock.Object);
-            var result = target.Run(new string[] { });
-
-            mock.Verify();
-            target.ExitCode.ShouldBe(ExitCodes.BreakThresholdViolated);
-            result.ShouldBe(ExitCodes.BreakThresholdViolated);
-        }
+        // [Fact]
+        // public void OnMutationScoreBelowThresholdBreak_ShouldReturn_ExitCodeBreakThresholdViolated()
```
Comment (richardwerkman) [inline src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs:101]: Why is this commented out?

**Pull request merged**